lib/db/schema/patch-067-068.sql

```diff
@@ -0,0 +1,155 @@
+ALTER TABLE `unverifiedTokens`
+ADD COLUMN `shortCodeHash` BINARY(32) DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+CREATE INDEX `unverifiedTokens_shortCodeHash`
+ON `unverifiedTokens` (`shortCodeHash`)
+ALGORITHM = INPLACE LOCK=NONE;
+
+ALTER TABLE `unverifiedTokens`
+ADD COLUMN `shortCodeExpiresAt` BIGINT UNSIGNED DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+CREATE PROCEDURE `verifyTokenShortCode_1` (
+  IN `shortCodeHashArg` BINARY(32),
+  IN `uidArg` BINARY(16)
+)
+BEGIN
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
```

Comment:
This exit-handler logic is only necessary if you're going to use a transaction, which actually it might be worth doing here anyway because we're modifying multiple tables.